AppointmentReminders.Web/Scripts/bootstrap-datetimepicker.js

```diff
@@ -0,0 +1,2426 @@
+/*! version : 4.14.30
```

Comment:
That could be an alternative, nonetheless I don't see a big impact in using a CDN or using a local copy of the library, I'll have a look in the extra dependencies in a while.